src/components/PopupContainer/PopupContainer.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react'
+import PropTypes from 'prop-types'
+import './PopupContainer.scss'
+
+function PopupContainer({ isOpen, onClose, popupName, element }) {
+  // Для закрытия попапа по клавише escape
+  useEffect(() => {
+    const close = e => {
+      if (e.keyCode === 27) {
```

Comment:
```suggestion
      if (e.key === 'Escape' || e.key === 'Esc') {
```
keyCode задепрекейчен, Escape юзается в обычных браузерах, Esc юзается в IE/Edge